jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -41,23 +53,27 @@ node(platform) {
             } else {
                 println('Debug Build - Using default developer signing key')
             }
-        }    
+        }
+           if (platform == 'ios') {
+            codeSign(
+                profileId: "${CODE_SIGN_PROFILE_ID}",
+                clean: CLEAN,
+                verify: true,
+                appPath: "platforms/ios/build/emulator/${PROJECT_NAME}.app"
+            )
+        }
     }
 
     stage("Archive") {
         if (platform == 'android') {
-            if (buildType == 'release') {
-                archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-release.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
-            } else {
-                archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-debug.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
-            }
+             if (buildType == 'release') {
+                 archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-release.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
```

Comment:
yes agreed. we did that with the one we tested on Friday. I will push the change here also